ROOTFS/etc/waggle/sanity/fatal/agent-present.test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Agent Present Test: Begin"
+
+expectedAgent=$(cat /etc/waggle/node_manifest.json | jq '[.nxagent | select(.present == true)] | length')
+
+if [[ $expectedAgent == 0 ]]; then
+    echo "Agent Present Test: No agent expected for this device PASS"
+    exit 0
+fi
+
+nmapOut=$(nmap -sP 10.31.81.1/24)
```

Comment:
The agent is set to come up as 10.31.81.3 via our dnsmasq rules.  maybe we want to test for specifically that?